sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3626,6 +3626,10 @@ _ZN2cl4sycl5eventC1Ev
 _ZN2cl4sycl5eventC2EP9_cl_eventRKNS0_7contextE
 _ZN2cl4sycl5eventC2ESt10shared_ptrINS0_6detail10event_implEE
 _ZN2cl4sycl5eventC2Ev
+_ZN2cl4sycl5INTEL15online_compilerILNS1_15source_languageE0EE7compileIJSt6vectorINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESaISC_EEEEES6_IhSaIhEERKSC_DpRKT_
```

Comment:
`online_compile_error` should probably also be added to the list of ABI-checked classes

Reply:
online_compile_error is defined only in the header right now and thus no need in ABI check for this class.

Reply:
you are right